docs/relying-parties/how-twos/google-iap.md

```diff
@@ -0,0 +1,109 @@
+   - We hope to simplify this process in SubPlat 3.0, expected in 2023; see this [Jira comment](https://mozilla-hub.atlassian.net/browse/PSP-64?focusedCommentId=627245) for more details.
+3. Add each Play Store `productId` as a comma-separated value to a new `playSkuIds` metadata property on the newly created plan.
+
+### Prevent Google IAP subscribers from double subscribing
+
+As noted previously, we don’t support upgrades or any plan changes for Google IAP subscribers. However, it is possible for a Google IAP subscriber to try to sign up again for your product on the SubPlat website. Follow these steps to prevent them from double subscribing.
+
+1. In the Stripe dashboard for the given environment (stage or production), remove any `productSet` properties that may have been previously set on the newly created product/plan from [Map Google IAP to Stripe plans](#map-google-iap-to-stripe-plans).
+2. Add a `productSet` metadata key with a value equal to the comma-separated list of all the unique `productSet`s for all your product’s plans in the environment to the new product referenced in Step 1. Without their own `productSet` specified, the new plan will inherit its product’s `productSet`.
+
+## SubPlat API Calls
+
+These steps can be done by a developer and in parallel with the steps in [Configuration](#configuration).
+
+As mentioned in the [Introduction](#introduction), SubPlat has read-only access to Play Store subscription information. We will store Google IAP subscribers' data in our system and update the information when we receive notifications from the Play Store server, broadcasting updates in real time to the relevant RPs.
+
+### Getting the current subscription status for a FxA user
+
+Assuming you are already integrated with FxA generally (see [Prerequisites](#prerequisites)), are receiving subscription updates for other types of subscriptions (e.g. for PayPal or Stripe), and you have completed the [mapping of Play Store to Stripe plans](#map-google-iap-to-stripe-plans), no further work is required to continue receiving these updates for Google IAP subscribers. You can also pull this information from FxA with the existing [`/account/profile`](/ecosystem-platform/api#tag/Account/operation/getAccountProfile) endpoint without any further changes.
+
+### Registering a Google IAP subscriber
+
+SubPlat needs to be able to associate a Google IAP subscription in Google's system to a specific FxA user. This is done by registering each Google IAP subscriber with the [`/subscriptions/iap/play-token/${appName}`](/ecosystem-platform/api#tag/Subscriptions/operation/postOauthSubscriptionsIapPlaytokenAppname) endpoint.
+
+By design, we allow only one FxA user per Play Store [`token`](https://developer.android.com/google/play/billing#tokens), which is how Google uniquely identifies the user's device when the subscription was purchased.
```

Comment:
This might be different for Google. Try checking their docs else ask Ben.

Reply:
Updated.